nvm.sh

```diff
@@ -1581,6 +1581,9 @@ nvm_install_binary() {
   local TMPDIR
   local VERSION_PATH
 
+  local NODE_OR_IOJS
+  [ "${FLAVOR}" = "node" ] && NODE_OR_IOJS="${FLAVOR}"
```

Comment:
if `set -e` is set, won't this line error out?

I'd prefer a proper `if` here.